frontend/src/components/common/Resource/ResourceTable.tsx

```diff
@@ -84,7 +80,7 @@ export interface ResourceTableProps<RowItem> {
   columns: (ResourceTableColumn<RowItem> | ColumnType)[];
   /** Show or hide row actions @default false*/
   enableRowActions?: boolean;
-  actions?: null | RowAction[];
+  actions?: null | HeaderAction[];
```

Comment:
I think we need to leave this with the previous type because plugins may be using it and it would mean a type break for them.